src/handler.rs

```diff
@@ -171,15 +170,16 @@ impl<I, E> fmt::Debug for Response<I, E> {
     fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
         let mut fmt = fmt.debug_struct("Response");
         match self.item {
-                ResponseTypeItem::Result(_) => fmt.field("item", &"Result(_)".to_string()),
-                ResponseTypeItem::Fut(_) => fmt.field("item", &"Fut(_)".to_string()),
-        }.finish()
+            ResponseTypeItem::Result(_) => fmt.field("item", &"Result(_)".to_string()),
+            ResponseTypeItem::Fut(_) => fmt.field("item", &"Fut(_)".to_string()),
+        }
+        .finish()
     }
 }
 
 impl<I, E> Response<I, E> {
     /// Creates an asynchronous response.
-    pub fn async<T>(fut: T) -> Self
+    pub fn r#async<T>(fut: T) -> Self
```

Comment:
Since it is for new crate version, I think it would be better to remove all `async` keyword usage and just use something like `future`

Reply:
good catch